lib/widgets/boolean_field.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_quest/widgets/title_text.dart';
+
+class AppSwitchField extends StatelessWidget {
+  final String title;
+  final Function(bool) onChanged;
+  final bool initial;
+
+  const AppSwitchField({
+    super.key,
+    required this.title,
+    required this.onChanged,
+    required this.initial,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        TitleText(title),
```

Comment:
Please merge the code from the main into this branch to see the latest changes implemented by **"Add Option for Null Values #7"** 
This issue abstracts the implementation of Title and now we only need to implement the actual widget.